routes/register.go

```diff
@@ -395,6 +395,10 @@ func Register(c Config) {
 					r.Use(ctx.RequireViewDataset)
 
 					// view only functions
+					// tab publications
+					r.With(ctx.SetNav("datasets"), ctx.SetSubNav("publications")).Get("/publications", datasetviewing.ShowPublications).Name("dataset_publications")
```

Comment:
you're setting Nav twice, it's already set higher up in the router hierarchy

Reply:
done